pulsar-client/src/test/java/org/apache/pulsar/client/impl/MessageTest.java

```diff
@@ -81,4 +81,15 @@ public void testTopicMessageImplNoReplicatedInfo() {
         assertFalse(topicMessage.isReplicated());
         assertNull(topicMessage.getReplicatedFrom());
     }
+
+    @Test
+    public void testMessageImplGetReaderSchema() {
+        MessageMetadata builder = new MessageMetadata();
+        builder.hasSchemaVersion();
```

Comment:
@Shawyeok - Why was this line necessary? It seems like it's not doing anything.

Reply:
Indeed, should I open a PR to remove this line?